python_modules/automation/automation_tests/test_repo.py

```diff
@@ -0,0 +1,16 @@
+import os
+import subprocess
+from pathlib import Path
+
+
+def test_all_libraries_register() -> None:
+    # attempt to ensure all libraries in the repository register with DagsterLibraryRegistry
+    register_call = "DagsterLibraryRegistry.register"
+
+    library_dir = Path(__file__).parents[2] / "libraries"
+    assert str(library_dir).endswith("dagster/python_modules/libraries")
+
+    for library in os.listdir(library_dir):
+        result = subprocess.run(["grep", register_call, (library_dir / library), "-r"])
+        if result.returncode != 0:
+            assert False, f"Dagster library {library} is missing call to {register_call}."
```

Comment:
have a better idea than this? I put it in automation instead of dagster since its a meta repo level test but can move it where ever

Reply:
nope I must have missed that

Reply:
to be clear I added this in response to your review - was a good call